### PR TITLE
Routing fixes for dev code.

### PR DESCRIPTION
**Pull request by qubyte**
⚠️ WIP ⚠️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,12 +5,33 @@
...
+                if (!existingJob) {
+                    return null;
+                }
+
+                try {
+                    const { token, jobId, serviceId, supportEmail, local, domain } = JSON.parse(existingJob);
+                    const sdk = createEndUserSdk({ token, jobId, serviceId });
+                    const job = await sdk.getJob();
+
+                    if (job.status === 'success' || job.status === 'fail') {
```
Comment (muzafarova) [inline index.html]: Isn't `job.finishedAt` more future proof?

File: index.html [hunk 2]
```diff
@@ -5,12 +5,33 @@
...
         <link href="/src/css/index.css" rel="stylesheet">
         <script type="module">
-            window.location.hash = '';
-            localStorage.clear();
-
             import { createEndUserSdk } from '/web_modules/@ubio/sdk.js';
             import { createApp } from '/src/main.js';
 
+            async function getExisting() {
+                const existingJob = localStorage.getItem(`job-session`);
```
Comment (muzafarova) [inline index.html]: later in code it's `job-session-${domain}` https://github.com/universalbasket/sdk-application-bundle/pull/65/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR48 
intentional?

File: index.html (hunk 1 above)
Comment (qubyte) [reply index.html]: Good idea :+1:

File: index.html (hunk 2 above)
Comment (qubyte) [reply index.html]: I need to remove this. The problem is that storage elsewhere doesn't distinguish.

File: index.html (hunk 1 above)
Comment (qubyte) [reply index.html]: This is also a good idea for SDK. Job tracking automatically stopped for end states.

**Pull request merged**